### ISSUE TITLE
Trying to start navigation without sound by toggle the mute button on code, but seem to not work.

### ISSUE THREAD

**Issue by JFDionne** [mapbox/mapbox-navigation-android#1965]
**Android API:** 24
**Mapbox Navigation SDK version:** 0.39.0

### Steps to trigger behavior

 1. I'm using the Embedded navigationView
 2. Using this code before calling navigationView.startNavigation(options.build()):
```
		boolean isMuted = ((SoundButton) navigationView.retrieveSoundButton()).toggleMute();
		if (!isMuted)
			((SoundButton) navigationView.retrieveSoundButton()).toggleMute();
```

### Expected behavior
Starting navigation muted

### Actual behavior
I see the muted icon, but I still hear the sound.

full code:

```
		NavigationRoute.builder(getContext())
				.accessToken(Mapbox.getAccessToken())
				.origin(Point.fromLngLat(location.getLongitude(),location.getLatitude()))
				.destination(DESTINATION)
				.alternatives(true)
				.build()
				.getRoute(new SimplifiedCallback() {
					@Override
					public void onResponse(Call<DirectionsResponse> call, Response<DirectionsResponse> response) {
						DirectionsRoute directionsRoute = response.body().routes().get(0);

						boolean isMuted = ((SoundButton) 																		 
                                                navigationView.retrieveSoundButton()).toggleMute();
						if (!isMuted)
						    ((SoundButton) navigationView.retrieveSoundButton()).toggleMute();
					        }

					        NavigationViewOptions.Builder options =
					            NavigationViewOptions.builder()
					            .navigationListener(this)
					            .directionsRoute(directionsRoute)
					            .shouldSimulateRoute(true)
					            .progressChangeListener(this)
					            .instructionListListener(this)
					            .speechAnnouncementListener(this)
					            .bannerInstructionsListener(this)
					            .darkThemeResId(R.style.NavigationViewDark)
					            .offlineRoutingTilesPath(obtainOfflineDirectory())
					            .offlineRoutingTilesVersion(obtainOfflineTileVersion());

					        navigationView.startNavigation(options.build());
				});
```

**Comment by Guardiola31337** [comment]
WIP in https://github.com/mapbox/mapbox-navigation-android/pull/1970

**Comment by Guardiola31337** [comment]
@abhishek1508 is this going to be fixed in `1.0`? https://github.com/mapbox/mapbox-navigation-android/pull/1970 is pretty out of date at the moment.

cc @JunDai
